src/main/java/by/onliner/cucumber/steps/MainPageSteps.java

```diff
@@ -0,0 +1,28 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static by.onliner.core.utils.EqualsUtil.equalContains;
+
+public class MainPageSteps extends AbstractSteps{
+
+    @When("Click [О компании] link")
+    public void clickOnAboutCompanyLink() {
+        mainPage.getFooter().clickOnAboutCompanyLink();
+    }
+
+    @Then("Page url should contain {string}")
+    public void isPageUrlContainLink(String link) {
```

Comment:
I think this method can be moved to AbstractSteps as it can relate to all the pages, do you agree?

Reply:
Agree